src/dotnet-interactive-vscode/src/tests/unit/ipynbMetadata.test.ts

```diff
@@ -329,25 +329,51 @@ describe('ipynb metadata tests', () => {
     });
 
     describe('cell language selection', () => {
-        it(`cell language is first determined from cell metadata`, () => {
+        it(`cell language is first determined from cell text`, () => {
```

Comment:
The passive voice is a little funny alongside the "it..." style of the mocha tests. Alternatively...

```suggestion
        it(`sets the cell language based on the cell text`, () => {
```